api/v1beta2/authpolicy_types.go

```diff
@@ -197,7 +202,8 @@ func (s *AuthPolicyStatus) Equals(other *AuthPolicyStatus, logger logr.Logger) b
 	return true
 }
 
-var _ common.KuadrantPolicy = &AuthPolicy{}
+var _ utils.KuadrantPolicy = &AuthPolicy{}
+var _ utils.Referrer = &AuthPolicy{}
```

Comment:
Is "utils" the best package for these?

Reply:
Possibly not, I've added a kuadrant package to break up the `utils` package a bit, which might be a better package for these (some are required to prevent a cyclic import)